### PR TITLE
TA-106617

### PR DESCRIPTION
**Pull request by ashleykwtam**
Addresses [TSE-272](https://cockroachlabs.atlassian.net/browse/TSE-272):

Preview: https://deploy-preview-17628--cockroachdb-docs.netlify.app/docs/advisories/a106617

### CODE REVIEW

**Comment by ashleykwtam** [comment]
@jbowens Here is the draft of the TA, edited for language and links. Let me know if you're OK with this. v22.1.22 is going out today, would the TA go out after?

File: src/current/advisories/a106617.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+advisory_date: 2023-08-11
+docs_area: releases
+---
+
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+A CockroachDB store with [encryption-at-rest](../v23.1/security-reference/encryption.html#encryption-at-rest) enabled maintains per-file metadata necessary for file decryption within an append-only log. When this [log file](../v23.1/configure-logs.html) grows large, the log file is rotated by writing a snapshot of the metadata to a new, replacement log file. The introduction of this log file in [v21.2.0](../releases/v21.2.html) contains a bug in log rotation that could result in the omission of a single file’s metadata from the persisted log file. When a node encounters this bug, it continues to operate normally until the process is restarted. If the omitted file still exists after a process restart, the [`cockroach` process](../v23.1/cockroach-node.html) cannot decrypt or read the file, and the local store appears to be corrupted. This bug most frequently causes the `cockroach` process on a node to exit shortly after it starts. An error such as “`pebble/table: invalid table (bad magic number: 0x08cf6caea91de887)`” is logged.
```
Comment (jbowens) [inline src/current/advisories/a106617.md:15]: The log file referred to in "When this log file grows large," is a different kind of log file than the one described in `../v23.1/configure-logs.html`. The log files described in `../v23.1/configure-logs.html` are diagnostic, only intended for operators to examine, whereas the encryption-at-rest log file is binary and only intended for CockroachDB itself to read during start. I think we should remove the link on "log file."

**Comment by jbowens** [comment]
> Let me know if you're OK with this. v22.1.22 is going out today, would the TA go out after?

LGTM, besides one nit. I think publishing after v22.1.22 is published makes sense; I'm not sure what sequencing we've done for previous TAs.

File: src/current/advisories/a106617.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: Technical Advisory 106617
+advisory: A-106617
+summary: When the encryption-at-rest registry rolls over, the append-only log file contains a bug that could result in loss of the store. 
+toc: true
+affected_versions: v21.2.x, ≤ 22.1.21, ≤ 22.2.12, ≤ 23.1.7
+advisory_date: 2023-08-11
```
Comment (jbowens) [inline src/current/advisories/a106617.md:7]: nit: Should this be updated to 2023-08-14 if we publish today?

**Review by MichaelTrestman — APPROVED** [review]
LGTM

**Pull request merged**